grammars/keywords-ar.lark

```diff
@@ -1,10 +1,10 @@
-_PRINT:  ( "ـ"* "ق" "ـ"* "و" "ـ"* "ل" "ـ"* | "print") _SPACE?
+_PRINT: ("قول" | "print") _SPACE?
```

Comment:
This overrides the (manual) changes that we made in #2699 (which we don't want)

Since we are now automatically generating anyway... could we add "ـ"* in between all characters automatically in Arabic while generating?